sky/utils/controller_utils.py

```diff
@@ -180,7 +203,7 @@ def _get_cloud_dependencies_installation_commands(
     ]
     # k8s and ibm doesn't support open port and spot instance yet, so we don't
     # install them for either controller.
```

Comment:
Feeling like for now we could install dependencies for every cloud on the job controller. every cloud should support at least ondemand instance and should support managed ondemand jobs at least?